src/process_wu_map_data.R

```diff
@@ -0,0 +1,76 @@
+# Take data from Water Use team and reformat to what we need for the bubble map in the visualization
+# *** ONLY FOR THERMOELECTRIC (aka TE) RIGHT NOW ***
+
+# INPUT (unzipped data from `HUC12centroid_TE.7z` at ftp://ftpint.usgs.gov/private/wr/id/boise/Skinner/WU/)
+#   1. HUC12_TE_2015with.csv – comma-delimited file of withdrawal water use in m3/day by HUC12
+#   2. HUC12centroid_TEspt.json –geojson file of centroids for HUC12s with thermoelectric plant(s)
+# OUTPUT
+#   1. Long format geoJSON with thermoelectric water use values transformed into bubble radii (spatial 
+#      features duplicated for each timestep)
+#   2. Wide format geoJSON with thermoelectric water use values transformed into bubble radii (unique  
+#      spatial features with separate column for each timestep)
+
+# Currently a manual prep process: download, unzip, and move files listed
```

Comment:
I agree not worth using Jenkins at this point, but it would just be a few lines to fully automate the download/unzip for local running

Reply:
the only problem is that I don't know if the `ftp` they sent will be the data's forever home. Would like to wait and see

Reply:
but I guess, it's easy to just change the ftp address ...

Reply:
Oh wait, it is a 7z file and I can't script unzipping that. That's why I went the manual route.

Reply:
There is apparently an R package that uses a system library: https://stackoverflow.com/questions/16096192/how-to-programmatically-extract-unzip-a-7z-7-zip-file-with-r.  Unclear if the package is stale or not, and it does have system dependencies, so 🤷 if you want to bother or not

Reply:
I tried, but neither the `archive` package or the system command worked. Coded in with a check so that someone else could try.